app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -88,10 +88,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     mSwipesButton.setOnClickListener(this);
     mBrbButton.setOnClickListener(this);
 
+    // Resetting the buttons on page reload
+    changeButtonColor(R.color.blue, R.color.wash, mNorthButton);
+    changeButtonColor(R.color.blue, R.color.wash, mWestButton);
+    changeButtonColor(R.color.blue, R.color.wash, mCentralButton);
+    changeButtonColor(R.color.blue, R.color.wash, mSwipesButton);
+    changeButtonColor(R.color.blue, R.color.wash, mBrbButton);
+
```

Comment:
why not:

```
HashMap<String, Button> mButtons = new HashMap<>();

String[] viewIds = { R.id.northButton, R.id.westButton, ... };
for (String id : viewIds) {
  Button tempButton =  rootView.findViewById(R.id.brb);
  mButtons.put(id, tempButton);
  tempButton.setOnClickListener(this);
  changeButtonColor(R.color.blue, R.color.ash, tempButton);
}
```

Reply:
lol I had a similar comment: https://github.com/cuappdev/eatery-android/pull/99#pullrequestreview-210407329